frontend/src/features/admin-form/settings/SettingsEndPage.stories.tsx

```diff
@@ -16,15 +11,10 @@ import {
 import { SettingsEndPage } from './SettingsEndPage'
 
 const buildMswRoutes = ({
-  overrides,
   delay,
 }: {
-  overrides?: Partial<FormSettings>
   delay?: number | 'infinite'
-} = {}) => [
-  getAdminFormSettings({ overrides, delay }),
-  patchAdminFormSettings({ overrides }),
-]
+} = {}) => [getAdminForm({}, delay)]
```

Comment:
should we just  use `createFormBuilderMocks` instead? it already contains a handler for this new function you created.

Reply:
Okay, updated PR to use `createFormBuilderMocks`. I removed the `getAdminForm` implementation as well